internal/statemachine/classic_states.go

```diff
@@ -183,8 +184,12 @@ func (stateMachine *StateMachine) populateClassicRootfsContents() error {
 	}
 
 	if !strings.Contains(string(fstabBytes), "LABEL=writable") {
-		newFstab := []byte("LABEL=writable   /    ext4   defaults    0 0")
-		err := ioutilWriteFile(fstabPath, newFstab, 0644)
+		re := regexp.MustCompile(`(?m:^LABEL=\S+\s+/\s+(.*)$)`)
```

Comment:
Eh, annoying. How did we miss that? Guess there was just too much code to write and review!